### PR TITLE
Feat/events basics

### PR DESCRIPTION
**Pull request by JohanHjelsethStorstad**
THIS PR SHOULD BE MERGED AFTER #313.

This PR adds the basics of the event system, including event-models and the main event pages. Next up:
- Event tags (can be started now)
- Registrations (Needs money to a degree)
- event-auth (needs visibility)
- event migrations (can to a degree be written now)

### CODE REVIEW

**Comment by JohanHjelsethStorstad** [comment]
And no this PR is not 91 commits. Many of them come from #313 (not merged yet)

**Review by Paulijuz — APPROVED** [review]
Look good.

Pointed out a few spelling errors.

Also, I noticed that when for example setting the start date after the end date when creating a event the client side simply gives "unkown error". This is because the error message from the server error isn't sent to the client. This will probably be fixed in the future, but couldn't another approach be to validate the data with a transform with zod? We already have error message support with it and isn't this type of validation the exact use case for zod?

Anyways, I'll approve this PR and you can fix whatever you seem fit (and I'll approve again) or you can just merge.

File: src/services/events/create.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+export const create = ServiceMethodHandler({
+    withData: true,
+    validation: createEventValidation,
+    handler: async (prisma, _, data) => {
+        const cmsParagraph = await createCmsParagraph({ name: uuid() })
+        const cmsImage = await createCmsImage({ name: uuid() })
+
+        if (data.eventStart > data.eventEnd) {
+            throw new ServerError('BAD PARAMETERS', 'Event må jo strate før den slutter')
```
[the remark this answers is not in the document]
Comment (Paulijuz) [reply src/services/events/create.ts:19]: **starte

lol

[the remark this answers is not in the document]
Comment (JohanHjelsethStorstad) [reply src/services/events/create.ts:19]: I'll resolve this on next event-pr

**Pull request merged**